### DISCUSSION TITLE
error when trying to play a playlist  "I could not find "playlist.txt" in the Playlists folder."

### DISCUSSION THREAD

**Discussion by cte12312** [jagrosh/MusicBot#1401] [Troubleshooting]
Hi 
I am new to this subject so pls help me.
I created a playlist but when i try to play it it say  "I could not find "playlist.txt" in the Playlists folder."
when i checked the folder i saw the txt  file in it.
![Screenshot 2023-07-08 223515](https://github.com/jagrosh/MusicBot/assets/99129411/b902d42f-4864-41b7-89f8-239e74ceb000)

thank you in advance

**Comment by Svindle** [comment]
Playlist names are case sensitive. If you asked it to play Playlist, it won't find it since your playlist.txt is all lower case.

**Comment by TheHeiki** [comment]
saving the txt file encoding as UTF-8 fixed it for me
